test/testdrive/protobuf-import.td

```diff
@@ -0,0 +1,105 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Test that Protobuf imports are handled correctly.
+
+$ set importee-schema
+syntax = "proto3";
+
+message Importee1 {
+    bool b = 1;
+}
+
+message Importee2 {
+    int32 i = 1;
+}
+
+
+$ set importer-schema
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
```

Comment:
What do you think of moving this import to one of the messages in `importee-schema`? I think it could be good to exercise chains of references that need to be imported from CSR -- so we're confident that we're not just importing top level references

Reply:
Ooh good idea. I moved the timestamp import as you suggest, and replaced it with a useless import of an empty file, to test that we continue to handle schemas with multiple references correctly.